EarTrumpet/App.xaml.cs

```diff
@@ -166,7 +166,9 @@ private void OnCriticalFontLoadFailure()
                 Environment.Exit(0);
             }).Start();
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
the visual studio in my pc treat this warning as an error

Reply:
Yes, it works in Release mode. You can just comment this out for now.  I was going to replace it with new Mutex().WaitOne(); but need to test that it works as expected. We should also fix the Debug/Release to use the same /W4 or 'convert warnings to errors' semantics, so I left it until we get the next version out.

Reply:
I ended up using ```new AutoResetEvent(false).WaitOne();```, so you can undo this part.

Reply:
It has rebased @davuxcom